### PR TITLE
initial jscs work and formatting

### LINKED ISSUE #43: use jscs pesets
use the OLP jscs presets defined [here](/OpenLightingProject/javascript-style)

### PR DESCRIPTION
**Pull request by daveol**
fixes #43 
TODO:
 - [x] fix formatting

### CODE REVIEW

**Comment by nomis52** [comment]
Dangling underscores?

**Comment by daveol** [comment]
yeah I should probably disable it for that file

File: README.md [hunk 1]
```diff
@@ -4,24 +4,12 @@ This is the codebase for the rdm.openlighting.org website.
 
 If you're intested in how PID definitions work, see https://wiki.openlighting.org/index.php/RDM_PID_Definitions
 
-## Development Setup
+## Get dependencies
```
Comment (nomis52) [inline README.md]: This is reverting all the README changes.

**Comment by nomis52** [comment]
Looks find apart from the README change.

**Pull request merged**